tests/commands/test-sdk.js

```diff
@@ -777,10 +722,9 @@ describe('sdk', () => {
 					}
 
 					logger.calls[0].should.eql([ 'banner', undefined ]);
-					logger.calls[1].should.eql([ 'error', 'Invalid Titanium SDK "9.1.0.GA"\n' ]);
-					// it suggests installed sdks
-					logger.calls[2].should.eql([ 'log', 'Did you mean this?' ]);
-					logger.calls[3].should.eql([ 'log', `    ${'9.2.0.GA'.cyan}` ]);
+					logger.calls[2].should.eql([ 'error', 'Invalid Titanium SDK "9.1.0.GA"' ]);
+					// logger.calls[3].should.eql([ 'log', 'Did you mean this?' ]);
```

Comment:
How come these are commented out?

Reply:
it didn't show up in my tests but I can check it again :+1:

Reply:
even the server doesn't show that https://github.com/tidev/titanium/actions/runs/3292077256/jobs/5427012669#step:5:40